src/tensor_ops.jl

```diff
@@ -656,15 +656,18 @@ function Base.cross{T}(::Vec{1, T}, ::Vec{1, T})
 end
 
 # Symmetry checks
-function Base.issymmetric{dim}(t::Tensor{2, dim})
-    N = n_components(Tensor{2, dim})
-    rows = Int(N^(1/2))
-    @inbounds for row in 1:rows, col in row:rows
-        if t[row, col] != t[col, row]
-            return false
-        end
+@inline Base.issymmetric(t::Tensor{2, 1}) = true
```

Comment:
I wonder if some stuff would be simpler by just explicitly writing out the expression like this for all the dimensions. At least for the symmetric stuff.